tests/foreman/cli/test_activationkey.py

```diff
@@ -1581,3 +1581,65 @@ def test_positive_invalid_release_version(module_sca_manifest_org, module_target
         }
     )
     assert update_ak[0]['message'] == 'Activation key updated.'
+
+
+# -------------------------- MULTI-CV SCENARIOS -------------------------
+def test_positive_multi_cv_info(
+    session_multicv_sat, session_multicv_org, session_multicv_default_ak
+):
+    """Verify that multi-env AK display into hammer activation-key info commands output
+
+    :id: 6a1c3189-74f9-4a54-8579-f3b045870cd9
+
+    :steps:
+        1. Create two lifecycle environments and two content views, publish/promote to respective lce
+        2. Create activation key and update ak with multiple content view environments
+        3. Check ak info displays 'Multi Content View Environment' and 'Content View Environments'
+
+    :expectedresults: AK info displays 'Multi Content View Environment' and 'Content View Environments'
+
+    :CaseImportance: Medium
+
+    :CaseComponent: ActivationKeys
+
+    :team: Phoenix-subscriptions
+
+    :parametrized: No
```

Comment:
```suggestion
```
We could inherit the first two fields from the module (since the values are the same we don't need to redefine them) and `paramtrized: No` should be the default.